### ISSUE TITLE
Any contribution guidelines for this repo?

### ISSUE THREAD

**Issue by nomanoff** [ixartz/astro-boilerplate-components#1]
I was using the Atro Boilerplate (https://github.com/ixartz/Astro-boilerplate) and have noticed that the components do not take any style props. This was also mentioned in an issue (https://github.com/ixartz/Astro-boilerplate/issues/7). As a reply it was mentioned that the pull requests are welcome for this repo. So, I was curious if I could help with that. I am new to astro and typescript and I really wanna contribute to open-sourse. So, is there anything that I can help you guys? I am acutally looking for a first issue, because I haven't done any open-source contributions before. I would be really happy for this oportunity to help and also grow my own skills.

Thank you.

**Comment by ixartz** [comment]
@nomanoff Thank you for your help, all help are welcoming. Based on your comment, you don't have any experience in open-source contribution, what I can suggest you can start small by working on the "Adding extra Categories Tags breaks the UI fluidity on mobile apps." from this GitHub issue: https://github.com/ixartz/Astro-boilerplate/issues/7.

Then, you can tackle the second one.

Let me know what do you think.

**Comment by nomanoff** [comment]
Awesome. Thank you for the oportunity. I will check that issue out then.

**Comment by nomanoff** [comment]
I have some questions. How do you test the components while developing? I heard it is done with tools like storybook but don't see it in the dependencies so I wanted to know how you guys did it? Also, are there any rules or convetions I need to follow to contribute to this library? Thank you.

**Comment by ixartz** [comment]
Thank you for asking.

You can test the components with the main project with `npm link`. So, you need to work with both projects.

The rules and convention is already enforced by ESLint with Husky. It'll automatically run the linter before each commit.

**Comment by nomanoff** [comment]
Okay, I got it. Thank you

**Comment by nomanoff** [comment]
Good day. I have fixed the "Adding extra Categories Tags breaks the UI fluidity on mobile apps" issue with the "astro-boilerplate-components". However when I tried to test the package locally I am getting an error inside "astro-boilerplate" project. So, here is the what I did: 
1. I cloned this package and installed npm and then ran `npm link`.  After that I wrote the bug fix.
2. I cloned "astro-boilerplate" project and installed npm. To test the components with project I ran `npm link astro-boilerplate-components`.  Everything was fine. But, when I run the `npm run dev` inside the main project I got 
error saying `Failed to resolve entry for package "astro-boilerplate-components". The package may have incorrect main/module/exports specified in its package.json: Failed to resolve entry for package "astro-boilerplate-components". The package may have incorrect main/module/exports specified in its package.json.` 

Why do you think I got this error to run npm package locally? Did I follow wrong steps?

**Comment by nomanoff** [comment]
The fix is ready. I isoleted the `Project.tsx` component, which was responsible for the bug, and directly used it inside the main project. It was an easy fix with `flex-wrap`. It is working well. However, I wasn't able to test the whole component library due to the above mentioned error. 
So, if you want I can make the pull request so you can test it too. What do you think?

Thank you for your time.

**Comment by ixartz** [comment]
@nomanoff, Thank you for finding the solution. It looks good to me the solution with `flex-wrap`, the solution should work. You can make the pull request and I'll merge it.

If you are still interested by the project, we can remove `astro-boilerplate-components` and use only one project instead of two. It can be your next task?

**Comment by nomanoff** [comment]
Great then! I will make the pull request. Regarding your question, I am really interested in this project. Please give me some more deatails for the next task and I will try my best. 
Thank you for all the help.

**Comment by ixartz** [comment]
@nomanoff thank you for your first contribution, I just merged your pull request. There are a lot of work in this project, hope you'll grow your skill with more complex feature.

The idea is to remove `astro-boilerplate-components` because it's make hard for people to customize and also solve your issue "Failed to resolve entry".
You need to migrate the components and type files from `astro-boilerplate-components` to https://github.com/ixartz/Astro-boilerplate. Then, inside https://github.com/ixartz/Astro-boilerplate we can remove the dependency to `astro-boilerplate-components`. `astro-boilerplate-components` will no longer be used.

**Comment by nomanoff** [comment]
Thank you very much. I am really looking forward to working more with open-source. 
It means I need to create `components` folder inside [Astro-boilerplate](https://github.com/ixartz/Astro-boilerplate). And then write down all the `astro-boilerplate-components` there. I got it. 
I might take a little time to make the pull rquest because I am a student and a full-time dev. I am trying to work on this project as soon as I have free time. But, I will try my best. 
Thank you for all the guidence and help.

**Comment by ixartz** [comment]
Yes, creating the `components` will be perfect. I don't know but maybe you'll encounter some issue.

Sure, no problem, this is totally normal, we all have some commitment elsewhere.

**Comment by nomanoff** [comment]
Great then. I will work on that.

**Comment by m4g3s** [comment]
> 2\. Failed to resolve entry for package "astro-boilerplate-components". The package may have incorrect main/module/exports specified in its package.json: Failed to resolve entry for package "astro-boilerplate-components". The package may have incorrect main/module/exports specified in its package.json.

Hi, I'm wondering how you resolved this exactly? I'm experiencing the same error
